gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcTracingInterceptor.java

```diff
@@ -119,7 +121,7 @@ public void inboundTrailers(Metadata trailers) {
     /** Stream is closed. This will be called exactly once. */
     public void streamClosed(Status status) {
       logger.atInfo().log(
-          toJson(getRequestTrackingInfo("streamcloses()").put("status", status).build()));
+          "%s", toJson(getRequestTrackingInfo("streamcloses()").put("status", status).build()));
```

Comment:
why we need this?

Reply:
After update Error Prone static analysis enforces use of static constant as 1st argument of `log` method call.